foo_service.rb

```diff
@@ -0,0 +1,20 @@
+class CancelPendingDeposits < Service
```

Comment:
__[rubocop]__
* class definition in method body
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

<!-- Comment by [SideCI](https://sideci.com) -->